Formula/libcaption.rb

```diff
@@ -0,0 +1,41 @@
+class Libcaption < Formula
+  desc "Library for the creation and parsing of closed caption data"
+  homepage "https://github.com/szatmary/libcaption"
+
+  url "https://github.com/szatmary/libcaption/archive/0.7.tar.gz"
+  # version "0.7"
+  sha256 "125c9c55e1d5f0dc37ef151fa9583dd2fcfaefe7699d348c7292e634859e527e"
+  revision 0
+
+  # --HEAD version
```

Comment:
HEAD option should not exist in new formula, please remove it

Reply:
Removed, as requested